wagtailio/features/models.py

```diff
@@ -43,6 +51,7 @@ def __str__(self):
         InlinePanel("bullets", label="Bullets"),
         ImageChooserPanel("screenshot"),
         FieldPanel("video_url"),
+        MediaChooserPanel('video'),
```

Comment:
I've opened an issue on wagtailmedia - https://github.com/torchbox/wagtailmedia/issues/133- to allow more targeted choosers

Reply:
That would be awesome, thanks!